$NetBSD: patch-mkspecs_common_gcc-base.conf,v 1.1 2021/06/18 02:21:32 thor Exp $

Do not break -march=native builds with certain gcc versions. See
https://bugreports.qt.io/browse/QTBUG-71564 for detail and upstream
unwillingness.

--- ./mkspecs/common/gcc-base.conf.orig        2019-06-13 13:36:03.554816463 +0000
+++ ./mkspecs/common/gcc-base.conf
@@ -109,7 +109,8 @@ QMAKE_CFLAGS_MIPS_DSP  += -mdsp
 QMAKE_CFLAGS_MIPS_DSPR2 += -mdspr2
 
 # -march=haswell is supported as of GCC 4.9 and Clang 3.6
-QMAKE_CFLAGS_ARCH_HASWELL           = -march=core-avx2
+# https://bugreports.qt.io/browse/QTBUG-71564
+QMAKE_CFLAGS_ARCH_HASWELL           = -mavx2
 
 # Wrapper tools that understand .o/.a files with GIMPLE instead of machine code
 QMAKE_AR_LTCG           = gcc-ar cqs
